shared/actions/chat.js

```diff
@@ -472,6 +474,11 @@ function * _loadMoreMessages (): SagaGenerator<any, any> {
 
   let next
   if (oldConversationState) {
+    if (action.payload.onlyIfUnloaded && oldConversationState.get('paginationNext')) {
```

Comment:
only load more when we intend to actually load more. else we want to load the initial screen